src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <DiscordRPC/discord_register.h>
+#include <DiscordRPC/discord_rpc.h>
+#include <Windows.h>
+
```

Comment:
Don't pay for what you don't use.
No need to include files which are not used. And you should be especially careful with `windows.h` and not including it everywhere.
```suggestion
```